### PR TITLE
collectd: add write_prometheus plugin

### PR DESCRIPTION
**Pull request by felix**

### CODE REVIEW

File: srcpkgs/collectd/template [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+ $(vopt_enable prometheus write_prometheus)"
 conf_files="/etc/collectd.conf"
 hostmakedepends="automake pkg-config libtool libltdl-devel glib-devel"
 makedepends="libcurl-devel iptables-devel libressl-devel libsensors-devel
  libesmtp-devel zlib-devel yajl-devel python3-devel libxml2-devel net-snmp-devel
  libgcrypt-devel libmysqlclient-devel libltdl-devel libvirt-devel
- $(vopt_if notify libnotify-devel) $(vopt_if rrdtool rrdtool-devel)"
+ $(vopt_if notify libnotify-devel) $(vopt_if rrdtool rrdtool-devel)
+ $(vopt_if prometheus "protobuf-c-devel libmicrohttpd-devel")"
+depends="$(vopt_if prometheus protobuf-c libmicrohttpd)"
```
Comment (ericonr) [inline srcpkgs/collectd/template:19]: Aren't these libraries detected automatically? And multiple items like these should be in single quotes: `'protobuf-c libmicrohhtpd')"`

Comment (felix) [reply srcpkgs/collectd/template:19]: I thought the point was to ensure the dependent libraries are present if that option is selected?

Comment (ericonr) [reply srcpkgs/collectd/template:19]: The idea is that when building with this option, the final binaries will be dynamically linked to these libraries, meaning that the build system automatically detects them.

**Comment by ericonr** [comment]
Any reason this couldn't be enabled by default?

**Comment by felix** [comment]
Perhaps the following reasons:
- It depends on/installs extra libraries.
- collectd + prometheus is perhaps not a common combination for monitoring??

**Pull request merged**